src/booleans.js

```diff
@@ -1,62 +1,109 @@
 function negate(a) {
-  // your code here
-};
+  if (a === false) {
+    return true;
+  }
+  return false;
+}
 
 function both(a, b) {
-  // your code here
-};
+  if (a === true && b === true) {
+    return true;
+  }
+  return false;
+}
 
 function either(a, b) {
-  // your code here
-};
+  if (a === true || b === true) {
+    return true;
+  }
+  return false;
+}
 
 function none(a, b) {
-  // your code here
-};
+  if (a === false && b === false) {
+    return true;
+  }
+  return false;
```

Comment:
As before, this can be simplified. Remember the not operator (!) will invert the boolean value of whatever it prepends, so in the case of `a` being true, `!a` will evaluate to false. You can check if both values are false using:
```
  return !a && !b;
```